webpack.config.js

```diff
@@ -2,22 +2,15 @@ const path = require('path');
 const bemLoader = require('bem-loader');
 const CollectBemAssetsPlugin = bemLoader.CollectBemAssetsPlugin;
 const fs = require('fs-extra');
-const bemxjst = require('bem-xjst');
+function wrapTemplates (code) {
```

Comment:
of course. it allows not to reevaluate templates in compile call: https://github.com/bem/bem-xjst/blob/master/lib/bemxjst/index.js#L120-L122
see also https://github.com/bem/bem-xjst/pull/285